multi-arch-builders/provisioning/ppc64le/variables.tf

```diff
@@ -0,0 +1,78 @@
+
+variable "ibmcloud_api_key" {
+    description = "Denotes the IBM Cloud API key to use"
+    default = ""
+}
+
+variable "ibmcloud_region" {
+    description = "Denotes which IBM Cloud region to connect to"
+    default     = "us-south"
+}
+
+#INSERTED FOR MULTI-ZONE REGION SUCH AS FRANKFURT
+
+variable "ibmcloud_zone" {
+    description = "Denotes which IBM Cloud zone to connect to - .i.e: eu-de-1 eu-de-2  us-south etc."
+    default     = "us-south"
+}
+
+# Got the ID from `ibmcloud resource service-instances --long field` command, refer GUID for the instance
+variable "power_instance_id" {
+    description = "Power Virtual Server instance ID associated with your IBM Cloud account (note that this is NOT the API key)"
+    default = "556eb201-32bf-4ae2-8ab5-dfd7bbe97789"
+}
+
+variable "memory" {
+    description = "Amount of memory (GB) to be allocated to the VM"
+    default     = "50"
+}
+
+variable "processors" {
+    description = "Number of virtual processors to allocate to the VM"
+    default     = "15"
+}
+
+variable "proc_type" {
+    description = "Processor type for the LPAR - shared/dedicated"
+    default     = "dedicated"
+}
+
+variable "ssh_key_name" {
+    description = "SSH key name in IBM Cloud to be used for SSH logins"
+    default     = ""
+}
+
+variable "shareable" {
+    description = "Should the data volume be shared or not - true/false"
+    default     = "true"
```

Comment:
does this enable the data volume to be shared with other instances? do we need that? what are the drawbacks of true versus false?

Reply:
Price I would say. We can change it to false as well